README.md

```diff
@@ -87,3 +90,17 @@ jobs:
 ```
 
 The output tags from the `meta` step can be used as destination tags for this github action.
+
+### Use a custom .docker/config.json file
+
+In case you're running on a custom GitHub runner and you docker config path is not standard:
```

Comment:
```suggestion
### Use a custom docker config file

The standard docker config path on GitHub runner is `/home/runner/.docker/config.json`. In case you're running on a custom GitHub runner, and your config path is not standard, then the `docker-config-path` can be used.
```

What do you think about modifying the section like this so users have a better idea about the docker config.

Reply:
Sounds good!